roles/homebrew/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+# based upon https://gist.github.com/mrlesmithjr/f3c15fdd53020a71f55c2032b8be2eda
+
+- name: Ensuring Homebrew Is Installed
+  stat:
+    path: /usr/local/bin/brew
+  register: homebrew_check
+
+- name: Installing Homebrew
+  shell: /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+  when:
+    - not homebrew_check.stat.exists
+
+- name: Updating Homebrew
+  homebrew:
+    update_homebrew: true
+  when: homebrew_check.stat.exists
+
+- name: Upgrading Homebrew Packages
+  homebrew:
+    upgrade_all: true
+  register: result
+  until: result is successful
+  when: homebrew_check.stat.exists
+
+- name: Installing Homebrew Packages
+  homebrew:
+    name:
+      - opam
+      - pkg-config
```

Comment:
Add `tmux` here for running worker session?

Reply:
Let's switch to `launchctl` instead.